pkg/app/web/src/pages/settings/piped/index.tsx

```diff
@@ -182,36 +159,14 @@ export const SettingsPipedPage: FC = memo(function SettingsPipedPage() {
             </TableHead>
             <TableBody>
               {pipeds.map((piped) => (
-                <TableRow
-                  key={`pipe-${piped.id}`}
-                  className={clsx({ [classes.disabledItem]: piped.disabled })}
-                >
-                  <TableCell>
-                    <Typography variant="subtitle2">
-                      {`${piped.name} (${piped.id.slice(0, 8)})`}
-                    </Typography>
-                  </TableCell>
-                  <TableCell>{piped.version}</TableCell>
-                  <TableCell>
-                    <Typography variant="body2" color="textSecondary">
-                      {piped.desc}
-                    </Typography>
-                  </TableCell>
-                  <TableCell>
-                    {piped.startedAt === 0
-                      ? "Not Yet Started"
-                      : dayjs(piped.startedAt * 1000).fromNow()}
-                  </TableCell>
-                  <TableCell align="right">
-                    <IconButton
-                      edge="end"
-                      aria-label="open menu"
-                      onClick={(e) => handleMenuOpen(e, piped)}
-                    >
-                      <MoreVertIcon />
-                    </IconButton>
-                  </TableCell>
-                </TableRow>
+                <PipedTableRow
```

Comment:
nits, prefix `Piped` is redundant to me 👀

Reply:
@khanhtc1202 
If there were no `Piped`, wouldn't it be hard to distinguish it from the `TableRow` in `material-ui`?

Reply:
ops, I see 👀 In that case, maybe my suggestion is: rename the component to `TableRow` and in the calling side
```typescript
import { TableRow as PipedTableRow } from "./piped-table-row";
```
but in the second thought, It is quite laborious to do so, so the current implementation is LGTM 👍 
I'm going to merge this PR 🏃‍♂️